pkg/dryrun/validation.go

```diff
@@ -0,0 +1,272 @@
+package dryrun
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/libopenstorage/operator/pkg/util"
+)
+
+var (
```

Comment:
Why we need to define twice in seperate files? Can we have one place instead?

Reply:
var is only used by the current file